pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -435,6 +455,7 @@ func configureCustomTags(hcmTracing *hpb.HttpConnectionManager_Tracing,
 	if features.EnableIstioTags {
 		defaultTags := defaultTags()
 		tags = append(tags, defaultTags...)
+		tags = append(tags, buildCustomTracingTags(metadata)...)
```

Comment:
nit: should we update `defaultTags()` to be `optionalPolicyTags()` (or similar) and rename `buildCustomTracingTags()` to `buildServiceTags()` (or similar) to make what is happening clearer?

Reply:
Yeah, good idea. Default is not appropriate here.

Reply:
Done